src/server/replica.cc

```diff
@@ -478,7 +500,8 @@ error_code Replica::InitiateDflySync() {
     return cntx_.ReportError(ec);
   }
 
-  LOG(INFO) << absl::StrCat("Started full sync with ", server().Description());
+  LOG(INFO) << absl::StrCat("Started ", partial_sync ? "partial" : "full", " sync with ",
```

Comment:
Not that it matters greatly, but why `StrCat()` and not just stream into the `LOG` like usual?

Reply:
:shrug: